app/views/short_links/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= simple_form_for(@short_link, html: { class: 'form-inline shortlink-check' } ) do |f| %>
+  <%= f.input(:link, class: 'shortlink-field inline', label: false) %>
+  <span>
+    <%= fa_icon('arrow-circle-right') %>
+  <%= f.input(:target, class: 'inline', label: false, as: :string) %>
+  <%= f.button :submit %>
+
+  <p class="shortlink-notice">
+    <%= t('short_links.already_exists') %>
+  </p>
+<% end %>
```

Comment:
If you feel that these views should be coded in Erb instead of Haml, then that's ok but don't forget to delete the Haml views.